### PR TITLE
Default scoreboard

### PR DESCRIPTION
**Pull request by ahmedr2001**
# Description

Fix "default-show-all-contestants-in-scoreboard" bug

image

Fix: #6398 

# Check list:

- [ ] The code follows the [guide of
      style](https://github.com/omegaup/omegaup/wiki/Coding-guidelines) from
      omegaUp.
- [ ] All tests were run and passed.
- [ ] If new functionality is being added, tests have been added.
- [ ] If the change is large (> 200 lines), try to split it into
      several pull requests. Preferably one for controllers + phpunit
      and then another for the interface.

### CODE REVIEW

**Comment by heduenas** [comment]
@pabo99 please review this PR.

File: frontend/www/js/omegaup/components/contest/NewForm.vue [hunk 1]
```diff
@@ -532,7 +533,9 @@ export default class NewForm extends Vue {
       penalty: this.penalty,
       scoreboard: this.scoreboard,
       penalty_type: this.penaltyType,
-      default_show_all_contestants_in_scoreboard: false,
+      default_show_all_contestants_in_scoreboard: this.update
+        ? this.defaultShowAllContestantsInScoreboard
+        : false,
```
[the remark this answers is not in the document]
Comment (ahmedr2001) [reply frontend/www/js/omegaup/components/contest/NewForm.vue]: Done

**Review by pabo99 — APPROVED** [review]
It LGTM!

Defer to @heduenas

**Comment by ahmedr2001** [comment]
Great. Got any recommendations for another issue to fix?

**Review by heduenas — APPROVED** [review]

**Comment by heduenas** [comment]
@ahmedr2001 Thanks for fixing this bug! 

I would recommend looking at the bugs in this list: https://github.com/omegaup/omegaup/milestone/99
Those have high priority. Beware that some of those are rather time-consuming, but other ones should be a just few hours or work.

**Pull request merged**